modules/skeleton/react/lib/skeleton.tsx

```diff
@@ -68,13 +76,8 @@ export default class Skeleton extends React.Component<SkeletonProps, SkeletonSta
     const topPosition = (-1 * (diagonal - height)) / 2;
 
     return (
-      <SkeletonContainer
-        aria-label={loadingAriaLabel}
-        aria-live={'polite'}
-        role={'status'}
-        ref={this.ref}
-        {...elemProps}
-      >
+      <SkeletonContainer ref={this.ref} {...elemProps}>
+        <AccessibleHide>{loadingAriaLabel}</AccessibleHide>
```

Comment:
This was called `loadingLabel` in `support/v3.x`, in this version it is called `loadingAriaLabel`